packages/devtools_app/lib/src/screens/vm_developer/vm_object_model.dart

```diff
@@ -208,7 +209,18 @@ class ScriptObject extends VmObject {
   SourceLocation? get _sourceLocation => null;
 
   @override
-  String? get name => null;
+  String? get name => (obj.uri ?? scriptRef?.uri)?.split('/').last;
+
+  late final DateTime? loadTime;
+
+  @override
+  Future<void> initialize() async {
+    await super.initialize();
+
+    if (obj.loadTime != null) {
```

Comment:
pull `obj.loadTime` into a local var so that it does not need to be null checked twice

Reply:
Done